spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
> but slightly more complex to consume (e.g., `literal.raw || ("'" + literal.value + "'")`)

No, `value` should never be used as fallback for `raw`. Please see my comments above about this and which way would be proper and work even with optional `raw` that you suggest.